apps/common/components/PageProgressBar.tsx

```diff
@@ -0,0 +1,28 @@
+import {useUI} from '@yearn-finance/web-lib/contexts/useUI';
+import {useClientEffect} from '@yearn-finance/web-lib/hooks';
+
+import type {ReactElement} from 'react';
+
+type TPageProgressBarProps = {
+  isLoading: boolean;
+}
+
+function PageProgressBar({isLoading}: TPageProgressBarProps): ReactElement {
+	const {onLoadStart, onLoadDone} = useUI();
+    
+	useClientEffect((): () => void => {
+		if (isLoading) {
+			onLoadStart();
+		} else {
+			onLoadDone();
+		}
```

Comment:
nit: I think a bit easier to read if
```suggestion
		isLoading ? onLoadStart() : onLoadDone();
```

Reply:
@Majorfi maybe worth to rename these in the web-lib, it feels strange that if `!isLoading` is `onLoadDone()` and when it's loading is `onLoadStart()`. Because we're missing at least a) the loading hasn't started yet; and b) the is current loading.

Also, @xgambitox I feel that this effect `PageProgressBar` would be better in the `pages/veyfi/index.tsx` as it's just returning a fragment. Could we just move the `useClientEffect` to `pages/veyfi/index.tsx` and remove this file altogether?

Reply:
Sounds good, just feels a bit strange to have a component just returning a fragment. Perhaps that component should still come from the web-lib, wdyt?